src/ZVal.h

```diff
@@ -10,6 +10,7 @@ namespace zeek
 	{
 
 class AddrVal;
+class EnumVal;
```

Comment:
How the heck did that build before?

Reply:
HAH yeah I wondered the same thing.  It's presumably because previously the only times `ZVal.h` was included, `Val.h` had already been included.  The new include for `Type.h` doesn't have that property, however (to avoid circularities).